### ISSUE TITLE
plugin enable operation broke with SSL error

### ISSUE THREAD

**Issue by wdurairaj** [hpe-storage/python-hpedockerplugin#401]
```
Complete log:
Complete log:

time="2018-11-08T11:45:26-05:00" level=info msg="2018-11-08 16:45:26.251 18 ERROR hpe3parclient.ssh [-] Error connecting via ssh: Error relocating /usr/lib/python3.6/site-packages/cryptography-2.2.2-py3.6-linux-x86_64.egg/cryptography/hazmat/bindings/_openssl.abi3.so: SSL_CTX_set_psk_client_callback: symbol not found\x1b[00m" plugin=4aeeba37201aa20b799882bf55ecacc3bb647b19ac9246069847d148afe241ef
Nov  8 11:45:26 cssosbe02-b12 dockerd[2142]: time="2018-11-08T11:45:26-05:00" level=info msg="2018-11-08 16:45:26.251 18 ERROR hpedockerplugin.volume_manager [-] ('hpeplugin_driver do_setup failed, error is: %s', 'Error connecting via ssh: Error relocating /usr/lib/python3.6/site-packages/cryptography-2.2.2-py3.6-linux-x86_64.egg/cryptography/hazmat/bindings/_openssl.abi3.so: SSL_CTX_set_psk_client_callback: symbol not found'): paramiko.ssh_exception.SSHException: Error connecting via ssh: Error relocating /usr/lib/python3.6/site-packages/cryptography-2.2.2-py3.6-linux-x86_64.egg/cryptography/hazmat/bindings/_openssl.abi3.so: SSL_CTX_set_psk_client_callback: symbol not found\x1b[00m" plugin=4aeeba37201aa20b799882bf55ecacc3bb647b19ac9246069847d148afe241ef
Nov  8 11:45:26 cssosbe02-b12 dockerd[2142]: time="2018-11-08T11:45:26-05:00" level=info msg="2018-11-08 16:45:26.254 18 INFO hpedockerplugin.volume_manager [-] Failed to initialize 3PAR driver for array: https://192.168.67.2:8080/api/v1!Exception: HPE Docker Volume plugin not ready: ('hpeplugin_driver do_setup failed, error is: %s', 'Error connecting via ssh: Error relocating /usr/lib/python3.6/site-packages/cryptography-2.2.2-py3.6-linux-x86_64.egg/cryptography/hazmat/bindings/_openssl.abi3.so: SSL_CTX_set_psk_client_callback: symbol not found')\x1b[00m" plugin=4aeeba37201aa20b799882bf55ecacc3bb647b19ac9246069847d148afe241ef
Nov  8 11:45:26 cssosbe02-b12 dockerd[2142]: time="2018-11-08T11:45:26-05:00" level=info msg="2018-11-08 16:45:26.254 18 ERROR hpedockerplugin.backend_orchestrator [-] INITIALIZING backend: DEFAULT FAILED Error: HPE Docker Volume Plugin Start Plugin Service Failed: Failed to initialize 3PAR driver for array: https://192.168.67.2:8080/api/v1!Exception: HPE Docker Volume plugin not ready: ('hpeplugin_driver do_setup failed, error is: %s', 'Error connecting via ssh: Error relocating /usr/lib/python3.6/site-packages/cryptography-2.2.2-py3.6-linux-x86_64.egg/cryptography/hazmat/bindings/_openssl.abi3.so: SSL_CTX_set_psk_client_callback: symbol not found'): hpedockerplugin.exception.HPEPluginStartPluginException: HPE Docker Volume Plugin Start Plugin Service Failed: Failed to initialize 3PAR driver for array: https://192.168.67.2:8080/api/v1!Exception: HPE Docker Volume plugin not ready: ('hpeplugin_driver do_setup failed, error is: %s', 'Error connecting via ssh: Error relocating /usr/lib/python3.6/site-packages/cryptography-2.2.2-py3.6-linux-x86_64.egg/cryptography/hazmat/bindings/_openssl.abi3.so: SSL_CTX_set_psk_client_callback: symbol not found')\x1b[00m" plugin=4aeeba37201aa20b799882bf55ecacc3bb647b19ac9246069847d148afe241ef
Nov  8 11:45:26 cssosbe02-b12 dockerd[2142]: time="2018-11-08T11:45:26-05:00" level=info msg="2018-11-08 16:45:26.263 18 ERROR hpedockerplugin.backend_orchestrator [-] ERROR: None of the backends could be initialized successfully. Please rectify the configuration entries in hpe.conf and retry enable.\x1b[00m" plugin=4aeeba37201aa20b799882bf55ecacc3bb647b19ac9246069847d148afe241ef
```

**Comment by wdurairaj** [comment]
Yes Imran. analysis on the openssl library version 1.1.1 is right. 

Specifying apk add openssl , should have ideally picked up the above version since in the alpine linux package manager i do see it's 1.1.1

https://pkgs.alpinelinux.org/packages?name=openssl&branch=edge

**Comment by wdurairaj** [comment]
Search for 'openssl' in the text box (Packages) of the above link. Or you could see https://pkgs.alpinelinux.org/package/edge/main/armhf/openssl

**Comment by wdurairaj** [comment]
Also we are restricting the cryptography==2.2.2 in the requirements-py3.txt which i'm thinking of putting in a free form (without the equality check for 2.2.2), so that it will move ahead and pick up the latest version of openssl

That is another thing which I will perform experimentation.

**Comment by wdurairaj** [comment]
marking this as FIXED based on CI job on CentOS -- http://16.184.15.6:8080/job/docker-3par-iscsi-plugin-ubuntu/291/console

**Comment by prablr79** [comment]
CI is passed hence closing this bug
